go/test/endtoend/vtgate/schematracker/loadkeyspace/schema_load_keyspace_test.go

```diff
@@ -19,85 +19,19 @@ package loadkeyspace
 import (
 	"os"
 	"path"
-	"strings"
 	"testing"
-	"time"
 
 	"github.com/stretchr/testify/require"
 
-	"vitess.io/vitess/go/test/endtoend/utils"
-
 	"vitess.io/vitess/go/test/endtoend/cluster"
 )
 
 var (
 	clusterInstance *cluster.LocalProcessCluster
 	hostname        = "localhost"
-	keyspaceName    = "ks"
 	cell            = "zone1"
-	sqlSchema       = `
-		create table vt_user (
-			id bigint,
-			name varchar(64),
-			primary key (id)
-		) Engine=InnoDB;
-			
-		create table main (
-			id bigint,
-			val varchar(128),
-			primary key(id)
-		) Engine=InnoDB;
-
-		create table test_table (
-			id bigint,
-			val varchar(128),
-			primary key(id)
-		) Engine=InnoDB;
-`
 )
 
-func TestLoadKeyspaceWithNoTablet(t *testing.T) {
```

Comment:
This test case was removed as discussed in support case  #18138

Reply:
Is an empty string an allowed value for `--tablet_types_to_wait`? Or do we validate and reject it and default to `PRIMARY, REPLICA`?

Reply:
What I understood that default is empty but it fails the validation, so at least one tablet type have to be provided.

Reply:
> What I understood that default is empty but it fails the validation, so at least one tablet type have to be provided.

Correct, the `vtgate` refuses to start with empty string. Furthermore, it validates that the string corresponds to a serving tablet type.